libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -210,127 +226,89 @@ RC_Channel::set_pwm_all(void)
 void
 RC_Channel::set_pwm_no_deadzone(int16_t pwm)
 {
-    _radio_in = pwm;
+    radio_in = pwm;
 
-    if (_type_in == RC_CHANNEL_TYPE_RANGE) {
-        _control_in = pwm_to_range_dz(0);
+    if (type_in == RC_CHANNEL_TYPE_RANGE) {
+        control_in = pwm_to_range_dz(0);
     } else {
-        //RC_CHANNEL_ANGLE, RC_CHANNEL_ANGLE_RAW
-        _control_in = pwm_to_angle_dz(0);
+        //RC_CHANNEL_ANGLE
+        control_in = pwm_to_angle_dz(0);
     }
 }
 
-// returns just the PWM without the offset from radio_min
-void
-RC_Channel::calc_pwm(void)
-{
-    if(_type_out == RC_CHANNEL_TYPE_RANGE) {
-        _pwm_out         = range_to_pwm();
-        _radio_out       = (_reverse >= 0) ? (_radio_min + _pwm_out) : (_radio_max - _pwm_out);
-
-    }else if(_type_out == RC_CHANNEL_TYPE_ANGLE_RAW) {
-        _pwm_out         = (float)_servo_out * 0.1f;
-        int16_t reverse_mul = (_reverse==-1?-1:1);
-        _radio_out       = (_pwm_out * reverse_mul) + _radio_trim;
-
-    }else{     // RC_CHANNEL_TYPE_ANGLE
-        _pwm_out         = angle_to_pwm();
-        _radio_out       = _pwm_out + _radio_trim;
-    }
-
-    _radio_out = constrain_int16(_radio_out, _radio_min.get(), _radio_max.get());
-}
-
-
 /*
   return the center stick position expressed as a control_in value
   used for thr_mid in copter
  */
-int16_t
-RC_Channel::get_control_mid() const {
-    if (_type_in == RC_CHANNEL_TYPE_RANGE) {
-        int16_t r_in = (_radio_min.get()+_radio_max.get())/2;
+int16_t RC_Channel::get_control_mid() const
+{
+    if (type_in == RC_CHANNEL_TYPE_RANGE) {
+        int16_t r_in = (radio_min.get() + radio_max.get())/2;
 
-        if (_reverse == -1) {
-            r_in = _radio_max.get() - (r_in - _radio_min.get());
+        if (reversed) {
+            r_in = radio_max.get() - (r_in - radio_min.get());
         }
 
-        int16_t radio_trim_low  = _radio_min + _dead_zone;
+        int16_t radio_trim_low  = radio_min + dead_zone;
 
-        return (_low_in + ((int32_t)(_high_in - _low_in) * (int32_t)(r_in - radio_trim_low)) / (int32_t)(_radio_max - radio_trim_low));
+        return (((int32_t)(high_in) * (int32_t)(r_in - radio_trim_low)) / (int32_t)(radio_max - radio_trim_low));
```

Comment:
I'm not sure these are equivalent.   you've removed a _low_in + and a - low_in from two different parts of the calculation, but one is external to the multiplication and division, and one is not.